sql/understanding-the-query-execution-plan.md

```diff
@@ -97,9 +97,9 @@ mysql> EXPLAIN ANALYZE SELECT count(*) FROM trips WHERE start_date BETWEEN '2017
 
 ### Introduction to task
 
-Currently, there are two types of task execution: cop tasks and root tasks. A cop task refers to a computing task that is executed using the coprocessor in TiKV. A root task refers to a computing task that is executed in TiDB.
+Currently, there are two types of task execution: cop tasks and root tasks. A cop task refers to a computing task that is executed using TiKV Coprocessor. A root task refers to a computing task that is executed in TiDB.
 
-One of the goals of SQL optimization is to push the calculation down to TiKV as much as possible. The coprocessor is able to assist in execution of SQL functions (both aggregate and scalar), SQL `LIMIT` operations, index scans, and table scans. All join operations, however, will be performed as root tasks.
+One of the goals of SQL optimization is to push the calculation down to TiKV as much as possible. Coprocessor is able to assist in execution of SQL functions (both aggregate and scalar), SQL `LIMIT` operations, index scans, and table scans. All join operations, however, will be performed as root tasks.
```

Comment:
I prefer the original here, or TiKV's coprocessor.  My english rationale is that the coprocessor is part of something, but not a whole unit by itself.

PTAL @Hoverbear